pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -72,16 +72,34 @@ type Filter struct {
 // +listMapKey:=name
 type Filters []Filter
 
-// Role is role of the service in Kubernetes.
-// +kubebuilder:validation:Enum=Node;node;Service;service;Pod;pod;Endpoints;endpoints;EndpointSlice;endpointslice;Ingress;ingress
```

Comment:
(nit) if we keep the validation marker at this place then we avoid the duplication below?

Reply:
I have done it, lets see if the tests pass.